server/util/redact/redact.go

```diff
@@ -0,0 +1,77 @@
+package redact
+
+import (
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
+	"github.com/golang/protobuf/proto"
+
+	bespb "github.com/buildbuddy-io/buildbuddy/proto/build_event_stream"
+)
+
+const (
+	RedactionFlagAPIKey = 1 << iota
+)
+
+func RedactAPIKey(ctx context.Context, env environment.Env, event *bespb.BuildEvent) error {
```

Comment:
I am more guilty than most of naming stuff like this -- but the "right" way to name this in go is probably to call this method APIKey() and then your code would read:

if err := redact.ApiKey(asdasd); err != nil {
return err
}

Reply:
sg, done. I think it looks kind of weird within the package itself, but it's probably better to make it more readable at the callsite.